lib/service_check.pm

```diff
@@ -337,7 +337,7 @@ sub check_services {
         my $srv_proc_name = $service->{$s}->{srv_proc_name};
         my $support_ver = $service->{$s}->{support_ver};
         my $service_type = 'Systemd';
-        next unless (($service->{$s}->{before_migration} eq 'PASS') && _is_applicable($srv_pkg_name));
+        next unless _is_applicable($srv_pkg_name);
```

Comment:
I suggestion use a switch here for chained jobs.

Reply:
Add comments why is_spvm need check.

Reply:
I agree with coolgw, please add some comments for the reason to add powervm as an exception.

Reply:
Please run a VR not on powerVM to prove this change won't affect other test.